test/data/gtou/correction-invoice.json

```diff
@@ -10,6 +10,9 @@
 	"doc": {
 		"$schema": "https://gobl.org/draft-0/bill/invoice",
 		"$regime": "DE",
+		"$addons": [
+			"eu-en16931-v2017"
```

Comment:
Are these still required?

Reply:
Not anymore, but I left them in a couple of examples for visibility